core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -1,703 +1,432 @@
+        <div class="donation-box-iframe">
+          <donation-box></donation-box>
         </div>
       </div>
     </div>
   </div>
 </div>
+
 <style>
-  .oppia-donate-asterisk-color {
-    color: #ff3a44;
-    font-weight: 700;
-  }
-  .oppia-donate-subscribe-button {
-    background: #00645c;
-    border-radius: 4px;
-    color: #fff;
-    font-family: 'Capriola', 'Roboto', Arial, sans-serif;
-    font-size: 16px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 20px;
-    width: 100%;
-  }
-  .oppia-donate-contributor-img {
-    max-width: 200px;
-  }
-  .oppia-donate-page-avatar-img {
-    margin-bottom: -53px;
-    margin-left: 70px;
-    position: relative;
-    z-index: 100;
-  }
-  .oppia-donate-page .carousel-control-prev-icon,
-  .oppia-donate-page .carousel-control-next-icon {
-    background-color: #094142;
-    background-image: none;
-    background-size: 100%, 100%;
-    border-radius: 50%;
-    height: 40px;
-    width: 40px;
+  .donate-page {
+    display: flex;
   }
 
-  .oppia-donate-page .carousel-control-next-icon:after {
-    color: #fff;
-    content: '>';
-    font-size: 25px;
+  .donate-content-container {
+    background-color: #fff;
+    display: flex;
+    justify-content: center;
   }
 
-  .oppia-donate-page .carousel-control-prev-icon:after {
-    color: #fff;
-    content: '<';
-    font-size: 25px;
+  .donate-content {
+    display: flex;
+    flex-direction: column;
+    max-width: 550px;
+    overflow: hidden;
+    width: 100%;
   }
 
-  .oppia-donate-student-feedback-text {
-    margin: 0 auto;
-    max-width: 800px;
-  }
-  .oppia-donate-page-banner {
-    background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 1)), url('/assets/images/donate/donate-page-banner.webp');
-    background-size: cover;
-    height: 350px;
+  @media (max-width: 1155px) {
+    .donate-content-container {
+      width: 100%;
+    }
+
+    .donate-content {
+      margin: 40px 8px;
+    }
+
+    .donate-content-padding {
+      padding: 0 16px 25px 16px;
+    }
+
+    .donate-highlights {
+      .donate-hightlights-tile {
+        .highlight-tile-img {
+          height: 96px;
+          max-width: fit-content;
+          width: auto;
+        }
+
+        > div {
+          padding: 8px 7px;
+
+          > p:first-child {
+            color: #00645c;
+            font-size: 14px;
+            font-weight: 700;
+            line-height: 16px;
+            margin-bottom: 5px;
+          }
+
+          > p:last-child {
+            color: #333;
+            font-size: 11px;
+            font-weight: 400;
+            line-height: 12px;
+          }
+        }
+      }
+    }
+
+    .donate-modal-button {
+      background: #00645c;
+      border-radius: 8px;
+      bottom: 8px;
+      box-shadow: 0 4px 4px 0 #00000040;
+      height: fit-content;
+      left: 50%;
+      padding: 15px 45px;
+      position: fixed;
+      transform: translateX(-50%);
+      width: fit-content;
+
+      > span {
```

Comment:
@kapilag96 I think this is causing the CSS issues -- mgiht need to separate it into a different rule. Ditto below.

(Not sure when nested CSS selectors got introduced, but I was testing with Chromium 98 -- quite old, admittedly! I did test with Chrome too and it worked there, so I think this is probably a browser compatibility thing.)

Reply:
That is correct. Apparently, firefox's ESR doesn't support nested css as well. I updated the website and tested it on firefox esr release. @seanlip can you please let me know if items are properly formatted in chromium now?

Reply:
Yup this works now, thanks @kapilag96! Just the CI error is left, then we can merge it.